mypy/typeshed/stdlib/json/__init__.pyi

```diff
@@ -52,14 +50,12 @@ def loads(
     parse_float: Callable[[str], Any] | None = None,
     parse_int: Callable[[str], Any] | None = None,
     parse_constant: Callable[[str], Any] | None = None,
-    object_pairs_hook: Callable[[list[tuple[Any, Any]]], Any] | None = None,
-    **kwds: Any,
+    object_pairs_hook: Callable[[list[tuple[object, object]]], object] | None = None,
```

Comment:
this one kinda melts my brain, but I think this should then also be 

```suggestion
    object_pairs_hook: Callable[[list[tuple[Never, Never]]], object] | None = None,
```

bpr seems to agree at least:
https://basedpyright.com/?pythonVersion=3.13&typeCheckingMode=all&code=GYJw9gtgBAxmA28CmMAuBLMA7AzgOgEMAjGKdCABzBFSgGEDFjkAaKAZSQEcBXJLGEgCwAKFCQoqAJ4V0WAOZlK1WgDkkANyQhRogCZJgUYAAoAHgC56jeMyQBte-HQ5U91DwrJ76rSDa%2B2gC6IWxgRABWKKhBbAD0AJQWolCpUCBIHiBYUGa6IvqGUAD68ibFlhzcfAIOHl4Orv5QTaFQiVAAtAB8UERgCMkiaemZPNl9A-DlZgn5osVQALzG5fJzIkA